stepfunctions/stepfunctions/functions/check_agent_status_1/index.py

```diff
@@ -0,0 +1,29 @@
+import boto3
+import json
+import os
+
+s3_client = boto3.client('s3')
+bedrock_agent = boto3.client('bedrock-agent')
+
+# from aws_lambda_powertools import Logger, Tracer
```

Comment:
For clean code, please remove those commented-out lines. But I think it's actually helpful to have the Logger from the powertool. And you can log `response` for any troubleshooting :). Not a blocking comment. I'll leave it to you to decide on the logging.